eZ/Publish/API/Repository/Tests/SearchServiceTest.php

```diff
@@ -546,6 +546,16 @@ function ($a, $b) {
                 ),
                 $fixtureDir . 'UserMetadata.php',
             ),
+            array(
+                array(
+                    'filter' => new Criterion\Ancestor(array('/1/5/44/45/')),
```

Comment:
Maybe change this to test `(array('/1/5/44/', '/1/5/44/45/')` which should produce same result right? _(more logic covered)_